llvm/include/llvm/Option/OptTable.h

```diff
@@ -53,10 +53,8 @@ class OptTable {
 public:
   /// Entry for a single option instance in the option data table.
   struct Info {
-    /// A null terminated array of prefix strings to apply to name while
-    /// matching.
-    ArrayRef<StringLiteral> Prefixes;
-    StringLiteral PrefixedName;
+    unsigned PrefixesOffset;
+    unsigned PrefixedNameOffset;
     const char *HelpText;
```

Comment:
Presumably applying similar techniques to HelpText and the remaining character pointers is a logical next step.

Reply:
Yep, I stopped at the "smallest" increment where it seemed self-cohesive.